arroyo/processing/strategies/batching.py

```diff
@@ -113,6 +113,7 @@ def __init__(
 
         self.__batch: Optional[Batch[TResult]] = None
         self.__closed = False
+        self.__flush_done: Optional[float] = None
```

Comment:
Should this also be time.time()?

Reply:
not really. We do not necessarily start submitting messages right after instantiation.

Reply:
Aren't we just skipping the recording of the first batch then?

For future batches it looks like we are setting this value on the prior call to `strategy.poll()`, which seems to happens pretty much after instantiation, even if no messages were submitted.

Reply:
well, this is a good point